### DISCUSSION TITLE
How do I set the ASG scale in and out settings?

### DISCUSSION THREAD

**Discussion by trudesea** [npalm/terraform-aws-gitlab-runner#1218] [Q&A] [answered]
So I just starting working with this yesterday.   I'm trying to setup a schedule for the ASG scale in/out.  

I see in the variables.tf file for the module: runner_schedule_config

`# Configure optional scale_out scheduled action
    scale_out_recurrence = "0 8 * * 1-5"
    scale_out_count      = 1 # Default for min_size, desired_capacity and max_size
    scale_out_time_zone  = "Etc/UTC"
    # Override using: scale_out_min_size, scale_out_desired_capacity, scale_out_max_size

    # Configure optional scale_in scheduled action
    scale_in_recurrence = "0 18 * * 1-5"
    scale_in_count      = 1 # Default for min_size, desired_capacity and max_size
    scale_in_time_zone  = "Etc/UTC"
    # Override using: scale_out_min_size, scale_out_desired_capacity, scale_out_max_size`

Where do I actually override these values?  Even if I set the runner_schedule_enable to true, I don't see any schedules on ASG in the AWS UI.  Is there an example of how to set these values, I know just enough about terraform to be dangerous

Thanks

**Comment by trudesea** [comment, marked as answer]
Never mind, figured it out, comments are confusing however

**Comment by kayman-mk** [comment]
@trudesea Would be nice if you could update the confusing parts to improve the docs.
